tests/unit/neptune/new/attributes/series/test_string_series.py

```diff
@@ -45,3 +64,23 @@ def test_log(self):
             values = list(var.fetch_values()["value"].array)
             expected = list(range(0, 5000))
             self.assertEqual(len(set(expected)), len(set(values)))
+
+    @patch("neptune.attributes.series.fetchable_series.MAX_FETCH_LIMIT", 7)
+    def test_custom_offset_limit(self):
+        total = 100
+        offset = 27
+        limit = 42
+        words = [fake.word() for _ in range(total)]
+        series = StringSeriesValues(
+            values=[StringPointValue(step=i, value=words[i], timestampMillis=i) for i in range(total)],
+            totalItemCount=total,
+        )
+        with self._exp() as exp:
+            with patch(
+                "neptune.internal.backends.neptune_backend_mock.NeptuneBackendMock.get_string_series_values",
+                new=self._get_string_series_values_dummy_impl(series),
```

Comment:
If this is turned to a function then we could patch it in the `@patch` decorator, right? Might be a little cleaner